limit-break/objdef.cpp

```diff
@@ -0,0 +1,27 @@
+#include "stdafx.h"
+#include "objdef.h"
+
+void Ring_Main_SkyDeck(ObjectMaster *a1) {
+	if (a1->Data1->Scale.x == -1)
+		a1->Data1->Scale.x = 4;
```

Comment:
Does this adversely affect behavior of the ring? It appears as though they're supposed to fall under certain conditions.

Reply:
Oh it doesn't, well then I have no idea what's wrong.